.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Python 3",
+    // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+    "image": "mcr.microsoft.com/devcontainers/python:1-3.11-bullseye",
```

Comment:
Do we want to try for 3.12? I believe that's the most recent version that still works with the current depdendencies